src/index.ts

```diff
@@ -30,5 +30,7 @@ export {
     ContractEventArgs,
     Web3Provider,
     ZeroExConfig,
-    TransactionReceipt,
+    TransactionReceiptWithDecodedLogs,
+    LogWithDecodedArgs,
+    DecodedLogArgs,
```

Comment:
please put the relevant ones in public types list in `website`.

Reply:
Will do after this is finalized and merged